PocketKit/Sources/PocketKit/Collections/CollectionViewController.swift

```diff
@@ -37,13 +62,23 @@ class CollectionViewController: UIViewController {
         model: CollectionViewModel
     ) {
         self.model = model
+        self.moreButtonItem = UIBarButtonItem(
```

Comment:
just curious: any reason we are instantiating this one directly while the other two are lazy initialized?

Reply:
will update this to be lazy